### PR TITLE
new stripe added

### PR DESCRIPTION
**Pull request by YashPatel1809**
new stripe added for payment gateway.

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -506,13 +563,6 @@ app.post("/checkout", async (req, res) => {
       paymentStatus: "Pending",
     });
 
-    // mark checked out cart items as inactive
```
Comment (hanish-con) [inline backend/index.js:509]: This is needed

File: backend/index.js [hunk 2]
```diff
@@ -405,7 +478,7 @@ app.post('/cart', async (req, res) => {
 // CART GET API
 app.get('/cart', async (req, res) => {
   try {
-    const cart = await Cart.find({ active: true });
```
Comment (hanish-con) [inline backend/index.js:408]: This is needed

File: backend/index.js [hunk 3]
```diff
@@ -447,23 +520,7 @@ app.delete('/cart/:productId', async (req, res) => {
...
-      res.status(500).json({ message: "Internal server error" });
-  }
-});
-
-// DELETE API for Cart
-app.delete('/remove-cart/:cartId', async (req, res) => {
-  const { cartId } = req.params;
-
-  try {
-      const deletedItem = await Cart.findOneAndUpdate({ _id: new mongoose.Types.ObjectId(cartId) }, { active: false}, { new: true});
```
Comment (hanish-con) [inline backend/index.js:466]: This is needed

File: backend/index.js [hunk 4]
```diff
@@ -614,9 +664,9 @@ app.get('/orders', async (req, res) => {
         },
       });
 
-    // if (orders.length === 0) {
```
Comment (hanish-con) [inline backend/index.js:617]: This is needed

File: frontend/src/App.tsx [hunk 5]
```diff
@@ -25,7 +25,9 @@ import ForgotPassword from "./pages/forgotpassword";
 import { SettingsOrderDetails } from "./pages/userprofile/order/order";
 import SettingsFavoriteProductsPage from "./pages/userprofile/favourites/favourite";
 
-
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js";
+const stripePromise = loadStripe("pk_test_51QMNsDEbIUYKEOl9F5f6497Nhqmo4SSN00IqCBBaGPWJ4gjV7k7g702RndLhoPIqyarmHrE0omnlcdTZfvJDPTty00nEoyDfmK");
```
Comment (hanish-con) [inline frontend/src/App.tsx:30]: Is this original key ?

**Review by hanish-con — CHANGES_REQUESTED** [review]

**Comment by YashPatel1809** [comment]
error

**Pull request closed without merging**